### ISSUE TITLE
"f" key cause hl-line highlighting to flash

### ISSUE THREAD

**Issue by Johnson-A** [syl20bnr/spacemacs#7923]
#### Description :octocat:
When pressing the "f" key, the current line which is highlighted with hl-line-mode will flash from highlighted->unhighlighted->highlighted.
No other key [a-Z] or [0-9] produces this behavior, even "F". I've tested in org-mode, emacs-lisp-mode, etc and it occurs everywhere.

#### Reproduction guide :beetle:
- Start Emacs
- With a buffer using hl-line-mode, press the "f" key and watch the current line flicker.

*Observed behaviour:* :eyes: :broken_heart:
Pressing the "f" key cause current line to flicker its highlighted face on and off.

*Expected behaviour:* :heart: :smile:
Pressing the "f" key only inserts the letter "f" and doesn't cause the current line highlighting to change

#### System Info :computer:
- OS: darwin
- Emacs: 25.1.1
- Spacemacs: 0.200.5
- Spacemacs branch: master (rev. 664ba6a)
- Graphic display: t
- Distribution: spacemacs
- Editing style: emacs
- Completion: helm
- Layers:
```elisp
(octave latex helm auto-completion better-defaults emacs-lisp git markdown spell-checking syntax-checking version-control)
```
- System configuration features: JPEG RSVG IMAGEMAGICK NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS NS

**Comment by bmag** [comment]
Can confirm, same thing happens when pressing `t`, `T`, and `F`. This is an upstream issue in Evil (or Emacs). I've been able to reproduce this behavior with stock Emacs, `evil-mode` and `global-hl-line-mode` enabled (but strangely not with the local `hl-line-mode`).

**Comment by Miciah** [comment]
Can you reproduce the problem on Emacs 26.1? I wonder whether the problem is the same as https://debbugs.gnu.org/cgi/bugreport.cgi?bug=23510, which was fixed in Emacs 26.1 ([commit](http://git.savannah.gnu.org/cgit/emacs.git/commit/?id=16be3e90545972dec16014253a843229d5bdf388)).
